### DISCUSSION TITLE
Notes

### DISCUSSION THREAD

**Discussion by iu14472** [tanujnotes/Olauncher#432] [Ideas]
![2](https://github.com/tanujnotes/Olauncher/assets/133242157/96dd75dc-21b7-4c9e-9a1f-a0716654f82e)

**Comment by DeadSecTr** [comment]
?

**Comment by iu14472** [comment]
The ability to take notes by swiping your finger over a free space on the screen.
A trace of a swipe shorter than a gesture remains. If the length of the swipe is sufficient to open the application or the phone curtain, then the trace is removed.
